react/Product.js

```diff
@@ -197,7 +198,7 @@ export const parseToJsonLD = ({
 
   const category = getCategoryName(product)
 
-  const gtin = selectedItem?.ean || null
+  const gtin = selectedItem?.[gtinValue]
```

Comment:
```suggestion
  const gtin = selectedItem?.[gtinValue] || null
```

I would rather the null value if there's no gtinValue (EAN or skuID) instead of undefined, since null is a valid value for not having any bar code or a valid value if the product is with wrong configs. Other option for this case is to remove gtin if any of those info are not available or use productID, which will always be available